lib/src/base-styles/_nx-tile.scss

```diff
@@ -135,11 +135,20 @@ $nx-tile-background-color: $nx-white;
 }
 
 // On pages with tiles with multiple sections these are lighter weight titles
-.nx-tile__section-header {
-  border-top: 1px solid #b7bfdb;
-  margin-top: $nx-spacing-l;
+.nx-tile-subsection {
+  @include container-vertical;
+  margin-top: $nx-spacing-xl;
   margin-bottom: $nx-spacing-md;
```

Comment:
What's the purpose of this smaller bottom margin? It just collapses into the larger top margin of the next section

Reply:
> What's the purpose of this smaller bottom margin? It just collapses into the larger top margin of the next section

It should be safe to remove: https://github.com/sonatype/sonatype-react-shared-components/pull/64/commits/8808c4962dcbee77e2175930c6705301822aacc1

Reply:
> It should be safe to remove: 8808c49

Safest probably to set it equal to the top margin